integrated_channels/moodle/exporters/learner_data.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+Learner data exporter for Enterprise Integrated Channel Moodle.
+"""
+
+from logging import getLogger
+
+from django.apps import apps
+
+from enterprise.api_client.discovery import get_course_catalog_api_service_client
+from integrated_channels.integrated_channel.exporters.learner_data import LearnerExporter
+from integrated_channels.utils import parse_datetime_to_epoch_millis
+
+LOGGER = getLogger(__name__)
+
+
+class MoodleLearnerExporter(LearnerExporter):
+    """
+    Class to provide a Moodle learner data transmission audit prepared for serialization.
+    """
+
+    def get_learner_data_records(
+            self,
+            enterprise_enrollment,
+            completed_date=None,
+            is_passing=False,
+            **kwargs
+    ):  # pylint: disable=arguments-differ,unused-argument
+        """
+        Return a MoodleLearnerDataTransmissionAudit with the given enrollment and course completion data.
+        If completed_date is None, then course completion has not been met.
+        If no remote ID can be found, return None.
+        """
+        completed_timestamp = None
+        if completed_date is not None:
+            completed_timestamp = parse_datetime_to_epoch_millis(completed_date)
+
+        if enterprise_enrollment.enterprise_customer_user.user_email is None:
+            LOGGER.debug(
```

Comment:
small but may look cleaner? I feel like since there are so many uses of `enterprise_enrollment.enterprise_customer_user`, a cached variable would be nice ...